crates/biome_js_analyze/src/lint/suspicious/no_explicit_any.rs

```diff
@@ -43,7 +43,7 @@ declare_rule! {
     ///
     /// ```ts
     /// class SomeClass<T extends any> {
-    ///   message: Array<Array<unknown>>;
```

Comment:
This is probably a result of `find and replace` that breaks the tests in the website repo:

https://github.com/biomejs/website/actions/runs/9109965581/job/25043948850#step:8:378

Reply:
You forgot to fix this one in 7bcc112 @ematipico